### DISCUSSION TITLE
git push over ssh

### DISCUSSION THREAD

**Discussion by badonders** [community/community#35088] [New to GitHub] [answered]
Hi all,
Hopefully you can help me out... Most likely a newbee question or lack with experience in ssh, but the following occurs...
When trying to push some content to a private repository, i receive the following message...

**ubuntu@terraform:~/terraform-aws$ git push origin main
ssh: Could not resolve hostname github.com:badonders: Name or service not known
fatal: Could not read from remote repository.**

Please make sure you have the correct access rights
and the repository exists.

Should something to do with access rights, but when trying to connect with ssh to github.com I receive the following message...

**ubuntu@terraform:~/terraform-aws$ ssh -T git@github.com
Hi badonders! You've successfully authenticated, but GitHub does not provide shell access.**

I've reading a lot about this error message, but can't find the right one...
Thanks in advance...

Bart

**Comment by antonblr** [comment]
@badonders could you provide output of `git remote -v` ?

**Comment by badonders** [comment, marked as answer]
ubuntu@terraform:~/terraform-aws$ git remote -v
origin	ssh://git@github.com:badonders/terraform.git (fetch)
origin	ssh://git@github.com:badonders/terraform.git (push)

**Reply by antonblr** [reply]
You don't need `ssh` - see [GitHub SSH url format](https://docs.github.com/en/get-started/getting-started-with-git/about-remote-repositories#about-remote-repositories).

Try updating your remote as follows:
```
$ git remote set-url origin git@github.com:badonders/terraform.git
$ <try pushing your changes>
```

**Reply by badonders** [reply]
Thanks. I can communicate with GitHub. Running into another issue, but I will figure that out, hopefully.

**Reply by antonblr** [reply]
Glad you moved forward @badonders! Please update the question status to not show it as "Unanswered".

**Reply by badonders** [reply]
kudos for @antonblr for answering this question...
